### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by Nisaba**

### CODE REVIEW

File: SwissBitcoinPayPaymentProvider.cs [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+        /// </summary>
+        /// <returns>Result</returns>
+        public async Task<bool> CanRePostRedirectPayment(PaymentTransaction paymentTransaction)
+        {
+            if (paymentTransaction == null)
+                throw new ArgumentNullException(nameof(paymentTransaction));
+
+            //ensure that at least 5 seconds passed after order is placed
+            //P.S. there's no any particular reason for that. we just do it
+            if ((DateTime.UtcNow - paymentTransaction.CreatedOnUtc).TotalSeconds < 15)
```
Comment (SwissBitcoinPay) [inline SwissBitcoinPayPaymentProvider.cs]: 5 seconds or 15 seconds ?

**Review by SwissBitcoinPay — CHANGES_REQUESTED** [review]
Only a few changes/questions

File: Services/SwissBitcoinPayService.cs [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+        public string CreateInvoice(SwissBitcoinPaySettings settings, PaymentDataModel paymentData)
+        {
+
+            try
+            {
+                var invoice = new SwissBitcoinPayInvoiceModel()
+				{
+                    title = paymentData.Description,
+                    description = $"{paymentData.BuyerName} | Order : {paymentData.OrderID} | Store : {paymentData.StoreID.ToString()}",
```
Comment (SwissBitcoinPay) [inline Services/SwissBitcoinPayService.cs]: I think to record the StoreID is not necessary.
```suggestion
                    description = $"{paymentData.BuyerName} | Order : {paymentData.OrderID}",

```

Comment (Nisaba) [reply Services/SwissBitcoinPayService.cs]: No, the Store ID is mandatory in the WebHook for GrandNode

Comment (SwissBitcoinPay) [reply Services/SwissBitcoinPayService.cs]: Then I suggest you pass this value in the `extra` field (for ex. as `storeId`), just like you do for `customNote`, so you can get it back in your webhook function via `jsonData.extra.storeId`

The storeId shouldn't be explicitly displayed to the user, it's not valuable.

**Pull request merged**